Tests.dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.10 AS build
```

Comment:
Google search indicates `Dockerfile.test` may be a better name instead of `Tests.dockerfile`

Reply:
Is this used in GitHub actions?

Reply:
not yet, it is initial proposal. will change that and update actions as well